sql/patch_109_110_c.sql

```diff
@@ -0,0 +1,27 @@
+-- See the NOTICE file distributed with this work for additional information
+-- regarding copyright ownership.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+# patch_105_106_b.sql
+#
+# Title: Widen the name field in ncbi_taxa_name.
+#
+# Description:
+#   Widen the name field in ncbi_taxa_name table to allow for longer values.
+
+ALTER TABLE ncbi_taxa_name MODIFY COLUMN name VARCHAR(500) NOT NULL;
+
+# Patch identifier
+INSERT INTO meta (species_id, meta_key, meta_value)
+  VALUES (NULL, 'patch', 'patch_109_110_b.sql|ncbi_taxa_name_varchar500');
```

Comment:
It looks like the patch name still has the clashing `b` suffix, both here and in the patched databases. But I'm pretty sure they can be edited directly like this without going through the whole patching process again.
```suggestion
  VALUES (NULL, 'patch', 'patch_109_110_c.sql|ncbi_taxa_name_varchar500');
```